langchain/chains/loading.py

```diff
@@ -138,19 +138,19 @@ def _load_map_reduce_documents_chain(
 
 
 def _load_llm_bash_chain(config: dict, **kwargs: Any) -> LLMBashChain:
-    if "llm" in config:
-        llm_config = config.pop("llm")
-        llm = load_llm_from_config(llm_config)
+    if "llm_chain" in config:
+        llm_chain_config = config.pop("llm_chain")
+        llm_chain = load_chain_from_config(llm_chain_config)
     elif "llm_path" in config:
```

Comment:
i think llm_path should be llm_chain_path

Reply:
fixed